book/website/community-handbook/accessibility/accessibility-alt-text.md

```diff
@@ -0,0 +1,123 @@
+(ch-accessibility-alttext)=
+# Alt text
+
+Alternative text (or alt text) are used for describing the appearance and function of an image on an HTML page for users who cannot see it. 
```

Comment:
```suggestion
Alternative text (or alt text) is used for describing the appearance and function of an image on an HTML page for users who cannot see it.
```

Reply:
This is a great edit - we'll add to our alt text page!